### PR TITLE
Openapi Workflow

### PR DESCRIPTION
**Pull request by acarnagey**
- Adds workflows to openapi paths
- Refactor paths into separate files
image
image

### CODE REVIEW

**Review by BenjaminMoe — APPROVED** [review]

**Comment by nissimsan** [comment]
@acarnagey , could you add some more comments on this PR, pls? It isn't entirely clear on the call what is the effects of this.

**Comment by nissimsan** [comment]
@acarnagey, suggestion to split this into two separate PRs.

**Comment by nissimsan** [comment]
Also, @acarnagey, we recently removed the RDF classes from the respec: 

image

Can you please remove that in the PR?

File: docs/openapi/paths/workflows.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
+/schemas/workflows/businesscard.yml:
```
Comment (OR13) [inline docs/openapi/paths/workflows.yml:1]: prefer kebab case here.

Comment (OR13) [inline docs/openapi/paths/workflows.yml:1]:
```suggestion
/schemas/workflows/business-card-workflow.yml:
```

**Comment by acarnagey** [comment]
Do not merge need to fix script that generates openapi.yml on npm i

**Review by OR13 — CHANGES_REQUESTED** [review]
Mark as draft, make CI pass, request a re-review

File: packages/traceability-schemas/services/schemas.js [hunk 2]
```diff
@@ -8,11 +8,22 @@ const yaml = require('js-yaml');
 const Ajv = require('ajv').default;
 const addFormats = require('ajv-formats').default;
 
+const schemaKeys = ['common', 'credentials', 'presentations', 'snippets'];
```
Comment (OR13) [inline packages/traceability-schemas/services/schemas.js:11]: these are "components", but that is a nit

**Comment by acarnagey** [comment]
Closing this one out, the allOf in paths was causing additional validation errors

**Pull request closed without merging**